open-metadata-resources/open-metadata-archives/open-metadata-types/src/main/java/org/odpi/openmetadata/opentypes/OpenMetadataTypesArchive.java

```diff
@@ -217,6 +218,1600 @@ private ClassificationDef addTemplateClassification()
+
+        properties.add(property);
+
+        final String attribute2Name            = "serverType";
+        final String attribute2Description     = "Type of software server.";
+        final String attribute2DescriptionGUID = null;
+
+        property = archiveHelper.getStringTypeDefAttribute(attribute2Name,
+                attribute2Description,
+                attribute2DescriptionGUID);
+
+        properties.add(property);
+
+        final String attribute3Name            = "version";
+        final String attribute3Description     = "Deprecated attribute. Use the serverVersion attribute to define the version number of software server.";
+        final String attribute3DescriptionGUID = null;
+
+        property = archiveHelper.getStringTypeDefAttribute(attribute3Name,
+                attribute3Description,
+                attribute3DescriptionGUID);
+        property.setAttributeStatus(TypeDefAttributeStatus.DEPRECATED_ATTRIBUTE);
+
+        properties.add(property);
+
+        final String attribute4Name            = "serverVersion";
+        final String attribute4Description     = "Version number of the software server.";
+        final String attribute4DescriptionGUID = null;
+
+        property = archiveHelper.getStringTypeDefAttribute(attribute4Name,
+                attribute4Description,
+                attribute4DescriptionGUID);
+
+        properties.add(property);
+
+
+        typeDefPatch.setPropertyDefinitions(properties);
+        return typeDefPatch;
+    }
+
+
+    /**
+     * Deprecate clashing properties and add new ones to replace them.
+     * @return the type def patch
+     * @see #updateSoftwareServerPlatformEntity()
+     */
+    private TypeDefPatch updateSoftwareServerCapabilityEntity()
+    {
+        /*
+         * Create the Patch
+         */
+        final String typeName = "SoftwareServerCapability";
+
+        TypeDefPatch  typeDefPatch = archiveBuilder.getPatchForType(typeName);
+
+        typeDefPatch.setUpdatedBy(originatorName);
+        typeDefPatch.setUpdateTime(creationDate);
+
+        /*
+         * Build the attributes
+         */
+        List<TypeDefAttribute> properties = new ArrayList<>();
+        TypeDefAttribute       property;
+
+        final String attribute1Name            = "type";
+        final String attribute1Description     = "Deprecated attribute. Use the capabilityType attribute to describe the type of software server capability.";
+        final String attribute1DescriptionGUID = null;
+
+        property = archiveHelper.getStringTypeDefAttribute(attribute1Name,
+                attribute1Description,
+                attribute1DescriptionGUID);
+        property.setAttributeStatus(TypeDefAttributeStatus.DEPRECATED_ATTRIBUTE);
+
+        properties.add(property);
+
+        final String attribute2Name            = "capabilityType";
+        final String attribute2Description     = "Type of the software server capability.";
```

Comment:
I expect this is a general comment for **type**.   There are many subtypes for software server capability - this type is supposed to refer to the implementation/deployment  (eg this is an Oracle RDBMS) rather than the open metadata type - maybe the description could make this clearer?

Reply:
These descriptions are all just copy / pasted from the original descriptions...  What about a general change to them along these lines?  "Type of implemented or deployed software server capability."

Reply:
I like it - we could then use the same attibure name?  say deployedInplementationType ?

Reply:
Done -- I've gone with `deployedImplementationType`